src/components/application_manager/test/commands/mobile/simple_response_commands_test.cc

```diff
@@ -152,7 +152,7 @@ MATCHER_P2(CheckMessageParams, success, result, "") {
       success == (*arg)[am::strings::msg_params][am::strings::success].asBool();
   const bool is_result_code_correct =
       result ==
-      static_cast<uint32_t>(
+      static_cast<int32_t>(
```

Comment:
@LuxoftAKutsan, I think it change can't help with logger. Please exclude this changes from commit.